### PR TITLE
Export reward draft2

### PR DESCRIPTION
**Pull request by ashisherc**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -110,10 +110,12 @@
     "cucumber": "git+https://github.com/SebastienGllmt/cucumber-js.git",
     "eslint": "7.10.0",
     "eslint-config-airbnb": "18.2.0",
+    "eslint-config-prettier": "^6.15.0",
```
Comment (SebastienGllmt) [inline package.json:113]: Historically we prefer to use strict versions (remove the `^`) and manually validate any version upgrade.

File: app/containers/wallet/WalletSummaryPage.js [hunk 2]
```diff
@@ -187,6 +190,35 @@ export default class WalletSummaryPage extends Component<InjectedOrGenerated<Gen
 
     const notification = this._getThisPageActiveNotification();
 
+    let exportDialog;
+
+    const delegationStore = this.generated.stores.delegation;
+    const delegationRequests = delegationStore.getDelegationRequests(publicDeriver);
+
+    if(!this.readyToExportHistory({ delegationRequests, publicDeriver })){
```
Comment (SebastienGllmt) [inline app/containers/wallet/WalletSummaryPage.js:198]: Usually we have spaces between ifs and braces
ex
```
if (foo) {
} else {
}
```

I think it should be a lint warning

**Review by SebastienGllmt — APPROVED** [review]

File: app/containers/wallet/WalletSummaryPage.js (hunk 2 above)
Comment (ashisherc) [reply app/containers/wallet/WalletSummaryPage.js:198]: weird that eslint isn't complaining about it, will check that out

File: package.json (hunk 1 above)
Comment (ashisherc) [reply package.json:113]: Sure

**Pull request closed without merging**